.github/workflows/preview.yaml

```diff
@@ -14,6 +14,7 @@ jobs:
       SENTRY_ORG: exactly
       SENTRY_PROJECT: webapp
       SENTRY_ENVIRONMENT: development
+      NEXT_PUBLIC_NETWORK: 5
```

Comment:
why do we need this here?

Reply:
it was missing in the CI check run, making the build step fail.
I still don't understand why it didn't fail before. @jgalat do you know?

Reply:
```
Error occurred prerendering page "/activity". Read more: https://nextjs.org/docs/messages/prerender-error
6577
Error: unknown network id: NaN
```

Reply:
For some reason this build step started failing some days ago due to this variable missing when doing a pre-render of the activity page where we have a throw if the variable is not set. I am not really sure why it started failing... we haven't upgraded Next or changed any deps.
Another way to fix this would be to add a fallback on `pages/activity.tsx`, which one 
Others builds (like e2e) set up that variable.